http-client-spi/src/main/java/software/amazon/awssdk/http/AbortableInputStream.java

```diff
@@ -65,4 +66,12 @@ public void abort() {
         abortable.abort();
     }
 
+    /**
+     * Access the underlying delegate stream, for testing purposes.
+     */
+    @SdkTestInternalApi
+    public InputStream delegate() {
+        return in;
+    }
```

Comment:
Note to reviewer: Our existing tests relied on reflection to access 'in', which does not work in JDK 17. Now, we expose it via a test API. This should be okay, since this is a protected (not public) API.